src/app/modules/dsp/components/ServiceTable.tsx

```diff
@@ -19,7 +19,7 @@ type Props = {
 };
 
 
-const columnWidths = [5, 25, 25, 25, 20];
+const columnWidths = [5, 33, 33, 33, 33];
```

Comment:
shouldn't these add up to 100?

Reply:
@cmichel21 for some wierd reason, when using 100% in total on Services table, it doesn't really fill all the table width (100% of container). Althoug DSPs tab looks just fine with total `columnWidths` of 100%. 

<img width="1090" alt="Screenshot 2019-06-22 at 16 58 11" src="https://user-images.githubusercontent.com/16082811/59964856-30f63280-950f-11e9-9066-a589872a7f28.png">
<img width="1069" alt="Screenshot 2019-06-22 at 16 58 04" src="https://user-images.githubusercontent.com/16082811/59964857-30f63280-950f-11e9-9759-85ed1fed393a.png">

Reply:
So using more than 100% in total for `columnWidths` for Services table solved the issue.. Investigated good amount of time to find a better solution but couldn't...

Reply:
Ok